charts/perses/values.yaml

```diff
@@ -191,4 +191,4 @@ datasources:
   #     plugin:
   #       kind: PrometheusDatasource
   #       spec:
-  #         directUrl: https://prometheus.demo.do.prometheus.io
```

Comment:
(nit) can we remove this change, just to keep the PR clean as possible?

Reply:
Sure, my editor added it I don't mind rolling back.